.github/workflows/ci.yml

```diff
@@ -26,20 +26,20 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
     with:
       envs: |
-        - linux: test-oldestdeps-cov-xdist
+        - linux: test-oldestdeps-cov
           python-version: 3.9
-        - linux: test-xdist
+        - linux: test
           python-version: 3.9
-        - linux: test-xdist
+        - linux: test
           python-version: 3.10
-        - linux: test-xdist
+        - linux: test
           python-version: 3.11
-        - macos: test-xdist
+        - macos: test
           python-version: 3.11
-        - linux: test-cov-xdist
+        - linux: test-cov
           coverage: codecov
   test_upstream:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
     with:
       envs: |
-        - linux: test-rad-xdist
+        - linux: test-rad
```

Comment:
Dropping `xdist` testing because with the additional tests in this PR there are two issues with `xdist` in github actions.

1. `xdist` is causing memory size collisions. In the sense that by default action runners have 7Gb of memory shared across the two cores. GitHub will kill the action if it attempts to use more than this amount of memory at once. Some of the existing tests (prior to this PR) do use over 4Gb (some of the tests in this PR use over 4Gb too). It is possible for changes in the test ordering to cause two of these "largish" memory tests to hit at the same time, resulting in the action being killed. This reordering is inevitable given that the tests are dispatched on a first-come basis to the two cores. So adding/reordering tests may in cause this to occur. This happens under some conditions with these additional tests mixed in.
2. Also `xdist` can cause the CI to take longer than necessary:

	- https://github.com/spacetelescope/roman_datamodels/actions/runs/5006108889/jobs/8970943733 is a run with xdist used (7min).
	- https://github.com/spacetelescope/roman_datamodels/actions/runs/5006147741/jobs/8971028814 is without xdist (4min).

Also, note that without `xdist` there is little difference between lengths of test runs:

- https://github.com/spacetelescope/roman_datamodels/actions/runs/5006371967/jobs/8971569618?pr=181 (the head of this PR's changes, without `xdist`) takes 4ish min.
- https://github.com/spacetelescope/roman_datamodels/actions/runs/5006325192/jobs/8971454680 (head of main, has `xdist`) takes 3.5ish min.

Given the additional tests in this PR this indicates that `xdist` is not really doing anything for the CI.

This is not to say that `xdist` won't help make them faster in some circumstances, or make the tests always fail due to memory issues. The failure is GitHub specific, and the negligible change in run time is down to overhead vs core scaling. Locally, and/or use on different action runners (like the Large ones or an internally hosted one) will likely benefit from `xdist` because of the greater number of cores available and larger memory limits.

So I believe this is a reasonable change under the circumstances.

Reply:
This sounds entirely reasonable to me. Should we also remove `-xdist` in `ci_cron.yaml` as well?